### PR TITLE
Final plots

### PR DESCRIPTION
**Pull request by sktollman**

### CODE REVIEW

**Comment by sktollman** [comment]
I branched off of the reorg branch, you can just look at the final commit. If this is refreshed once the other PR is merged then I think it'll only show the last commit.

**Comment by sktollman** [comment]
Actually closed reorg pr and we'll merge the two together in this pr

File: README.md [hunk 1]
```diff
@@ -42,28 +44,38 @@ To load modules, run `modprobe`. (eg: `sudo modprobe -a tcp_vegas`). You should
 
 To generate the graphs in our report, run:
 ```
-$ sh run_experiments.sh
+$ sh run_all.sh
```
Comment (alanefl) [inline README.md:47]: add step after this: after run_all.sh has finished its first master iteration (i.e. you see "MASTER ITERATION: 2") printed to the terminal, Ctrl+Z

Comment (sktollman) [reply README.md:47]: i changed it to run 5 times instead of 1000, do we still need this?

Comment (sktollman) [reply README.md:47]: or can we remove the loop so ctrl+z doesn't need to happen?

Comment (alanefl) [reply README.md:47]: You can remove the loop, and hardcode in results-1 at the bottom instead of results-$i

Comment (alanefl) [reply README.md:47]: And also logs-1, and remove the echo statement that says MASTER ITERATION

File: generate_plots.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# figure 1
+python reproduction/plotting/figure1_plot.py results-1/figure1.csv plots/figure1.svg
+# figure 2a
+python reproduction/plotting/figure2_plot.py results-1/figure2a.csv plots/figure2a.svg -o 2a -c -l -b
+# figure 2b
+python reproduction/plotting/figure2_plot.py results-1/figure2b.csv plots/figure2b.svg -o 2b -c -l
+# bothlinks
+python reproduction/plotting/figure2_plot.py results-1/bothlinks.csv plots/bothlinks.svg
+# figure 2a + extra protos
+python reproduction/plotting/figure2_plot.py results-1/figure2a.csv plots/figure2a.svg -o 2a -c
```
Comment (alanefl) [inline generate_plots.sh]: output to figure2a_extra.svg and figure2b_extra.svg for the bottom two

**Review by alanefl — APPROVED** [review]
One lingering comment on `generate_plots` and making figure1 work with tiny traces

**Pull request merged**